content/get-started/docker-concepts/the-basics/what-is-a-container.md

```diff
@@ -32,6 +32,8 @@ Here's what makes them awesome. Containers are:
 - Independent. Each container is independently managed. Deleting one container won't affect any others.
 - Portable. Containers can run anywhere! The container that runs on your development machine will work the same way in a data center or anywhere in the cloud!
 
+Note that "container" is an overloaded term. Whenever you use the word, check whether your audience uses the same definition.
+
```

Comment:
I don't think this is helpful. I think the only thing this does is add ambiguity; these docs are, as you say, authoritative.